libs/login/build.gradle

```diff
@@ -26,6 +35,22 @@ android {
     }
 }
 
+static def loadDeveloperProperties(File defaultsFile, File developerFile) {
```

Comment:
These methods are similar to how we handle secrets, but sharing methods between Gradle projects is not really easy, so to reduce complexity and because these methods are small I've decided to write them again